packages/xml-views-validation/test/validators/element/use-of-deprecated-class-spec.ts

```diff
@@ -0,0 +1,144 @@
+import { expect } from "chai";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import {
+  GEN_MODEL_TIMEOUT,
+  generateModel
+} from "@ui5-language-assistant/test-utils";
+import { testValidationsScenario } from "../../test-utils";
+import { validateUseOfDeprecatedClass } from "../../../src/validators/elements/use-of-deprecated-class";
+
+describe("the use of deprecated class validation", () => {
+  let ui5SemanticModel: UI5SemanticModel;
+
+  before(async function() {
+    this.timeout(GEN_MODEL_TIMEOUT);
+    ui5SemanticModel = await generateModel({ version: "1.74.0" });
+  });
+
+  context("true positive scenarios", () => {
```

Comment:
You didn't add a context for the positive tests for enum... please remove this (or add one to the other tests to be consistent)

Reply:
will do